frontend/src/app.jsx

```diff
@@ -6,8 +6,6 @@ export default function App() {
   const [tasks, setTasks] = useState([]);
   const [content, setContent] = useState("");
 
-  const [update, setUpdate] = useState(false);
-
```

Comment:
I see that you used this pattern:
- Create a state variable named `update` containing a boolean.
- Use that state variable as a dependency in the `useEffect()` that is responsible of fetching the tasks.
- Whenever you want to re-fetch the tasks to have the latest status, you change the value of `update` so the re-fetch happens again.

While this pattern works, it's not optimal, as it requires to re-fetch the tasks every time there's a change. And in the case of deletion, we can just take the tasks that are already in the `tasks` state variable and remove the one from the array that we just deleted, and then React will re-paint the UI without that task. Same effect, but we saved a fetch to the db ;)

In this PR you will see I removed the usage of this `update` variable and just removed the task from the `tasks` state variable instead.